packages/gatsby-plugin-postcss/src/__tests__/__snapshots__/gatsby-node.test.js.snap

```diff
@@ -190,9 +197,12 @@ exports[`gatsby-plugin-postcss Stage: build-html / PostCss options 1`] = `
                     Object {
                       "loader": "/resolved/path/postcss-loader",
                       "options": Object {
-                        "plugins": Array [
-                          "autoprefixer",
-                        ],
+                        "postcssOptions": Object {
+                          "plugins": Array [
+                            "autoprefixer",
+                          ],
+                          "sourceMap": false,
+                        },
                         "sourceMap": false,
```

Comment:
Wait, this doesn't actually look right. In the 4.0.0 release of postcss-loader, `sourceMap` was left alone at the top level and not put inside of the new `postcssOptions` object. Let me see why an extra `sourceMap` property is getting placed inside of `postcssOptions` in the tests.